code/modules/power/supermatter/supermatter.dm

```diff
@@ -355,10 +355,13 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		removed = new()
 
 	damage_archived = damage
-	if(!removed || !removed.total_moles() || isspaceturf(T)) //we're in space or there is no gas to process
+
+	//NSV13 - airless areas don't cause damage
+	if(isspaceturf(T)) //we're in space
 		if(takes_damage)
```

Comment:
Could you not just do something like...

takes_damage = (isspaceturf(T) && removed && removed.total_moles()) ? takes_damage : FALSE

Reply:
The takes_damage flag is only there for the "hugbox" and fake crystals, if I go flipping it on and off I'll probably break those.